File: ruby/add.rb
#add = -> (firstNumber, secondNumber) {firstNumber + secondNumber}
def add (firstNumber, secondNumber)
	return firstNumber + secondNumber
end

puts "Enter the first number: "
firstNumber = gets.to_i
puts "Enter the second number: "
secondNumber = gets.to_i
sum = add(firstNumber, secondNumber)
print "Sum of #{firstNumber} and #{secondNumber} is #{sum}."

File: rust/add.rs
fn  main() {
	print!("Enter first number: ");
	let firstNumber: i32 = read!();
	print!("Enter second number: ");
	let secondNumber: i32 = read!();
	print!("Sum of {} and {} is {}", firstNumber, secondNumber, add(firstNumber, secondNumber));


}

fn add(firstNumber, secondNumber){
	sum = firstNumber + secondNumber;
	return sum;
}
